packages/frontend/src/lib/ExampleCard.spec.ts

```diff
@@ -90,10 +91,16 @@ test('pullImage function is called when Pull image button is clicked', async ()
 
   // Find and click the "Pull image" button
   const pullButton = screen.getByTitle('Pull image');
+
+  // spy on telemetryLogUsage function
+  const spyOnLogUsage = vi.spyOn(bootcClient, 'telemetryLogUsage');
```

Comment:
You can probably do `vi.mocked(bootcClient.telemetryLogUsage)` and `expect(bootcClient.telemetryLogUsage).toHaveBeenCalled()`